src/commands/draft/draftCommands.ts

```diff
@@ -214,8 +214,8 @@ type DraftDependencies = {
 
 async function getGitHubAuthenticationSession(): Promise<Errorable<AuthenticationSession>> {
     try {
-        // No special scopes are required for GitHub - we are just listing repositories/branches.
-        const scopes: string[] = [];
+        // Repo scope required to see public/private repos.
```

Comment:
@ReinierCC looks good to me, can you provide the details of official documentation for these scopes would be nice to have for historical purpose. Also, previous developer had the scopes defined and removed it later here is the PR [link](https://github.com/Azure/vscode-aks-tools/pull/537#discussion_r1603326957), adding here for more context. I agree we need scopes repo here to make it work for forked repo's but we need documentation link. Thank you.

Reply:
@hsubramanianaks Thanks for that suggestion.

Here's the documentation on scopes: https://docs.github.com/en/apps/oauth-apps/building-oauth-apps/scopes-for-oauth-apps

Would this be the best place to document or should something like this be a comment next to the code?

Reply:
Comment will also be good, Thanks Reinier.

Reply:
Sounds good!